src/test/java/com/karankumar/bookproject/ui/goal/ReadingGoalViewTest.java

```diff
@@ -50,8 +50,7 @@
 import static com.karankumar.bookproject.backend.entity.ReadingGoal.GoalType.PAGES;
 import static com.karankumar.bookproject.backend.entity.ReadingGoal.GoalType.BOOKS;
 import static com.karankumar.bookproject.backend.goal.CalculateReadingGoal.howManyReadThisYear;
-import static com.karankumar.bookproject.ui.goal.ReadingGoalViewTestUtils.findHowManyBooksInReadShelfWithFinishDate;
-import static com.karankumar.bookproject.ui.goal.ReadingGoalViewTestUtils.findHowManyPagesInReadShelfWithFinishDate;
+import static com.karankumar.bookproject.utils.ReadingGoalTestUtils.*;
```

Comment:
We aren't using wildcardi imports (see our style guide), so could you please change this?

Reply:
hey, sorry, what is reason behind it? I don't see arguments behind it in the style-guide, would be nice to have small explanation because for some people is just matter of habit

Reply:
Sure, I've created a new issue for this (#390).

I'll take this discussion over to the #book-project Slack channel as it's better there (we can then have an open-ended discussion). I'm happy to change the style guide on any of the points if there are better arguments in favour of one approach over the existing